paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/EmbeddedConfirmationHelper.kt

```diff
@@ -55,6 +55,7 @@ internal class EmbeddedConfirmationHelper(
             initializationMode = loadedState.initializationMode,
             configuration = loadedState.configuration.asCommonConfiguration(),
             appearance = loadedState.configuration.appearance,
+            linkConfiguration = null,
```

Comment:
I think we do want to supply this. I'm happy to do it in a follow up though if you'd like. (I think it just comes from the loader `state.linkState?.configuration`).

Reply:
We do! I was just going to follow-up with it since this PR is already big!

Reply:
Works for me!